<!DOCTYPE html>
<html lang="en">
<head>
  <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-179848670-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-179848670-1');
</script>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
  <title>Iff</title>

  <!-- CSS  -->
  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
  <!-- Quantico Font -->
<link href="https://fonts.googleapis.com/css?family=Mulish" rel="stylesheet">

  <link href="css/materialize.css" type="text/css" rel="stylesheet" media="screen,projection"/>
  <link href="css/style.css" type="text/css" rel="stylesheet" media="screen,projection"/>
</head>
<body>
 
	<nav class="black" role="navigation">
	  <div class="nav-wrapper container"><a id="logo-container" href="#" class="brand-logo">if and only if (iff)</a>
<!--     <ul id="nav-mobile" class="right hide-on-med-and-down">
      <li><a href="aboutus.html">About Us</a></li>
    </ul>
 -->
    </div>
  </nav>
  <div class="section no-pad-bot" id="index-banner">
    <div class="container">
      <br><br>
      <h1 class="header center grey-text"> Quantum Bioinformatics for COVID-19 and beyond </h1>
      <div class="row center">
	      <h5 class="header col s12 light"> Literature analysis, protein binding and topological protein analysis, powered by quantum algorithms and accessible on quantum devices.</h5>
      </div>
      <div class="row center">
<!--         <a target="_untitled" href="https://forms.gle/3a64ixYnRVsW4bkt7" id="download-button" class="btn-large waves-effect waves-light black">Join Mailing List</a>
      </div> -->
      <br><br>

    </div>
  </div>


  <div class="container" >
    <div class="section">

      <!--   Icon Section   -->
      <div class="row">
        <div class="col s12 m6">
          <div class="icon-block">
            <h2 class="center black-text"><i class="material-icons">blur_on</i></h2>
	    <h5 class="center">At the atomic level</h5>

            <p class="light">Iff's protein pruning and binding technologies finds the best binding sites on a protein by comparing binding energies at every atom. Not only does this provide strong information for further kinetics modeling, but it also helps in the incorporation of subtler, electromagnetic effects between atoms. This also means that our tool can help complete protein-ligand and large protein-protein models with little setup time, which helps when working on alphacoronoviral proteins and their immune counterparties.</p>
          </div>
        </div>

        <div class="col s12 m6">
          <div class="icon-block">
            <h2 class="center black-text"><i class="material-icons">equalizer</i></h2>
	    <h5 class="center">Quantum Algorithms and Quantum Devices, Today</h5>

            <p class="light"> We model atoms on both real quantum devices and large scale cloud infrastructures utilizing CPUs and GPUs. We have written algorithms and workflows around various quantum processing devices and quantum simulators. This gives us flexibility between HPC, QPU, and other resources towards quick results for a fast evolving virus, like SARS-CoV-2.
            </p>
          </div>
        </div>

      </div>

    </div>
    <br><br>
  </div>
  <div class="container">
    <div class="section">

      <!--   Icon Section   -->
      <div class="row">
          <div class="icon-block">
      <h3 class="">Publications, Presentations, and Completed Projects</h5>

            <p class="light"> We have been working on sets of software tools and research papers to determine our technologies' strengths and weaknesses, as well as sharing information about our progress widely:
           </p>   

           <h5> <a href="https://meetings.aps.org/Meeting/FWS24/Session/J01.39">Differences between Dimerized and Monomeric ACE2 Models on Extracellular TMPRSS2 and SARS-CoV-2 Spike Wild Type Binding Using a SETS-QAOA-MaxCut and ZDOCK Workflow</a> </h5>
           <p> We compare two different ACE2 structural models as they relate to binding SARS-CoV-2 Spike and TMPRSS2, as evaluated by SETS-QAOA-MaxCut and ZDOCK, and find results in line with recent research on ACE2 dimerized model binding. </p>

           <h5> <a href="https://meetings.aps.org/Meeting/MAR24/Session/FF00.10">UC Davis Lung Day 2024: Comparing Computationally Folded Structures for Human Surfactant Protein-D (hSP-D)</a> </h5>
           <p>  We compare a few different computational protein structure prediction (PSP) software tools in their prediction of SP-D's structure as it relates to binding SARS-CoV-2 Spike, as evaluated by SETS-QAOA-MaxCut and ZDOCK. </p>

             <h5> <a href="https://meetings.aps.org/Meeting/MAR24/Session/FF00.10"> American Physics Society 2024 March Meeting: QM/MM Molecular Dynamics Run within Quantum and Tensor-Based Algorithms towards Topologically Inspired, Accurate Protein Folding</a> </h5>
           <p>  We prototype and present a QM/MM protein folding pipeline using existing molecular mechanics (MM) software and state-of-the-art Symmetry Adapted Petrubation Theory (SAPT) inspired Quantum Mechanics (QM) models based on 3 prominent basis sets. </p>


          <h5> <a href="https://doi.org/10.1164/ajrccm-conference.2023.207.1_MeetingAbstracts.A3966"> American Thoracic Society 2023: Surfactant Protein-D (SP-D) Binding Against the SARS-CoV-2 Spike (S) Protein Using Quantum- and Classical In-silico Modeling: Role of Glycosylation</a> </h5>
           <p>  We show how QAOA-MaxCut based protein pruning with protein docking shows different binding between monomeric and trimer SP-D structures and SARS-CoV-2 spike. </p>


          <h5> <a href="https://www.youtube.com/watch?v=Vv_dH8rJCtI&pp=ygUXcTJiIHNhbWFydGggc2FuZGVlcCBpZmY%3D"> Presentation by Samarth Sandeep, CEO, at Q2B Silicon Valley 2022 </a> </h5>
           <p>  We pitch our latest work and vision.</p>


           <h5> <a href="https://doi.org/10.52519/00063"> Presentation by Dr. Yan Leyfman at Leading Edge of Cancer Research Symposium, MD Anderson, 2022: Improving Healthcare Access: Using an Ontology Platform to Connect Bronchial-Associated Lymphoid Tissue Patients with COVID-19 to Clinical Trials </a> </h5>
           <p>  We develop a bioinformatics oriented ontology platform, ranging from biochemistry modeling with quantum algorithms to protein structure keywording, to develop a more thorough ontology of serine proteases within COVID-19-Induced Co-Morbid BALT Lymphoma.</p>

           <h5> <a href="https://doi.org/10.1182/blood-2022-159853"> American Society of Hematology 2022: The Development of an Ontology Platform to Detect Serine Protease Involvement in COVID-19-Induced Co-Morbid BALT Lymphoma</a> </h5>
           <p> We develop a bioinformatics oriented ontology platform, ranging from biochemistry modeling with quantum algorithms to protein structure keywording, to develop a more thorough ontology of serine proteases within COVID-19-Induced Co-Morbid BALT Lymphoma. </p>

              <h5> <a href="https://doi.org/10.1002/ajh.26736"> Leukemia, Lymphoma and Myeloma Congress (LLM) 2022: Developing an Ontology-Based Platform to Detect Serine Protease Involvement in BALT Lymphoma Patients with COVID-19 & Connect Them to Clinical Trials </a> </h5>
           <p> We develop a bioinformatics oriented ontology platform to connect co-morbid BALT Lymphoma and COVID-19 patients to relevant clinical trials. (section P-019) </p>

            <h5> <a href="https://doi.org/10.1002/ajh.26736"> Winner of Best Abstract in Lymphoma at the Leukemia, Lymphoma and Myeloma Congress (LLM) 2022: Developing a Proteomic Framework of Oncopathologies for Rare Hematological Malignancies, like T Lymphomas, through Protein-Protein Interactions to Effectively Connect Patients to Real-World Clinical Trials </a> </h5>
           <p>  We develop a bioinformatics oriented ontology platform, ranging from biochemistry modeling with quantum algorithms to protein structure keywording, to connect T-Cell Lymphoma patients to relevant clinical trials. (section P-018)</p>

           <h5> <a href="https://doi.org/10.3389/fimmu.2022.945317"> SP-A binding to the SARS-CoV-2 spike protein using hybrid quantum and classical in silico modeling and molecular pruning by Quantum Approximate Optimization Algorithm (QAOA) Based MaxCut with ZDOCK</a> </h5>
           <p> Published in the <i> Frontiers in Immunology </i> journal's "Updates on the Role of Surfactant Proteins A and D in Innate Immune Responses" collection in September 2022, we use a QAOA-MaxCut based algorithm to aid in protein pruning prior to protein docking with ZDOCK and show binding of SP-A to SARS-CoV-2 Spike on the S2 subunit.</p>


           <h5> <a href="https://doi.org/10.1200/JCO.2022.40.16_suppl.e18560"> American Society of Clinical Oncology (ASCO) 2022: Tackling healthcare inequities through a social media-based platform to connect solid tumor patients with COVID-19 to clinical trials.</a> </h5>
           <p> To determine the relationship solid tumors and COVID-19, ontologies, which are groupings of terms and related identifiers, such as genes, were created for general search terms, utilizing the Human Phenotype Ontology. They were then combined with “COVID-19” and used as search terms in Twitter’s Standard Search tool. The keywords with the most matches were then queried through clinicaltrials.gov and European Bioinformatics Institute’s (EBI) Protein Search Tool to find relevant clinical trials and proteins. Finally, the proteins found by the EBI protein search were run through the SwissModel Tool to find relevant protein structures before being used in binding Iff Technologies' protein binding tool, which provides K values related to 50% inhibition for each medication or immunotherapy. This produced a set of disease-specific keywords that are related to top tweets, clinical trials, protein structures, and binding concentration values in relevant biomolecular pathways for the keyword set “Tumor COVID-19”. </p>

           <h5> <a href="https://doi.org/10.1182/blood-2021-151709"> American Society of Hematology 2021: Social Media Validation for Pre-Screened Ontologies As a Driver for Biomolecular Interaction Significance in COVID-19 and Hematological Malignancies </a> </h5>
           <p> With the rise of social media use during the COVID-19 pandemic, impressions from online content can affect behavioral changes resulting in exacerbating disparities in care. Thus, there exists a need to utilize social media platforms, like Twitter, to help augment preparedness, especially at the intersection between oncology and COVID-19, where tweets could help hint at potential biomolecular interactions.
To address this, a study was developed to assess relationship and ontologies on the interaction between hematological malignancies and COVID-19 on Twitter. </p>
      
           <h5> <a href="https://github.com/IffTech/GSG-Morph"> G/SG Morph </a> </h5>
           <p> The Graph/Subgraph Isomorphism Library for Quantum Annealers, based off of Calude, Dinneen, and Hua’s paper <a href="https://doi.org/10.1016/j.tcs.2017.04.016"> “QUBO formulations for the graph isomorphism problem and related problems” </a>. This library was recently <a href="https://www.cs.auckland.ac.nz/research/groups/CDMTCS/researchreports/publication-list-bydate.php?selected-date=2016>">cited </a> (entry 499, update.pdf) in an update to the University of Auckland CDMTCS (Centre for Discrete Mathematics and Theoretical Computer Science) report of the same title and authorship as the previously mentioned paper. </p>
            <h5> <a href="https://github.com/IffTech/Boson-Sampling"> Boson Sampling Library </a> </h5>
           <p> A library to better understand Aaronson-Arkhipov Boson Sampling. The Boson Sampling Library contains dedicated functions for calculating the submatrix of Interferometer matrices as well as the photon detection probabilities with the help of Xanadu's <a href="https://the-walrus.readthedocs.io/en/latest/"> The Walrus </a>  library. </p>
            <h5> <a href="https://www.atsjournals.org/doi/pdf/10.1164/ajrccm-conference.2021.203.1_MeetingAbstracts.A3802">American Thoracic Society Presentation: In Silico Comparison of SP-A and ACE2 Binding to the SARS-CoV-2 Spike
Protein Using Quantum Computing </a> </h5>
           <p> An investigation into the potential binding of Surfactant Protein-A (SP-A) and the SARS CoV-2 spike protein that utilized Polar+'s protein pruning technology to find top binding spots on both proteins before binding analysis with ZDOCK, completed in collaboration with <a href="https://health.ucdavis.edu/leadership/bios/haczku_bio.html">UC Davis Medical Center's Lung Biology Center </a>.</p>
          <h5> <a href="https://www.atsjournals.org/doi/pdf/10.1164/ajrccm-conference.2021.203.1_MeetingAbstracts.A3770">American Thoracic Society Presentation:  Role of Surfactant Protein D (SP-D) in COVID-19</a> </h5>
           <p> An investigation into the potential binding of Surfactant Protein-D (SP-D) and the SARS CoV-2 spike protein that utilized Polar+'s protein pruning technology to find top binding spots on both proteins before binding analysis with ZDOCK, completed in collaboration with <a href="https://health.ucdavis.edu/leadership/bios/haczku_bio.html">UC Davis Medical Center's Lung Biology Center </a>. This work also featured analysis into the RNA prevalence of SARS CoV-2 in mice after SP-D was removed, as well as SP-D prevalence in COVID patients at the UC Davis Medical Center in order to further verify or refute computational binding results.</p>
           <h5> <a href="https://web.archive.org/web/20230320124955/https://www.ept.ca/2021/02/xanadu-and-cmc-microsystems-to-collaborate-on-quantum-computing/"> Iff Selected for Xanadu and CMC Quantum Sandbox</a> </h5>
            <p> We have been selected to complete chemical modeling techninques using Xanadu's X8 photonic quantum device and accompanying software, StrawberryFields. </p>


           <h5> <a href="https://www.biorxiv.org/content/10.1101/2020.12.11.420968v1">Quantum Approximated Graph Cutting: A Rapid Replacement for T-REMD?</a></h5>
           <p> An investigation into the potential binding of Surfactant Protein-A (SP-D) and the SARS CoV-2 spike that utilized Polar+'s protein pruning technology on the Rigetti Aspen to find top binding spots on both proteins before binding analysis with ZDOCK, completed in collaboration with the <a href="http://bioinformatics.am/staff/">Bioinformatics Group of the International Scientific-Education Center</a> at the National Academy of Science, Republic of Armenia. This work also featured Temperature Replica Exchange Molecular Dymanics (T-REMD) from GROMACS running on the BRIDGES and JUEWELS supercomputing clusters, and a completely classical version of the Polar+ protein pruning software that utilized Goemans-Williamson's MaxCut algorithm to serve as points of comparison against results from the quantum algorithm running classically and running on the Aspen device.</p>
          <h5> <a href="https://chemrxiv.org/engage/chemrxiv/article-details/60c7494e702a9bd73c18b10b">Energetics Based Modeling of Hydroxychloroquine and Azithromycin Binding to the SARS-CoV-2 Spike (S)Protein - ACE2 Complex</a></h5>
           <p> An investigation looking to determine if there was a potential binding interaction between the hydroxychloroquine and azithromycin drug combo and the SARS CoV-2 spike - ACE2 complex that utilized Polar+'s protein pruning technology to find top binding spots on both proteins before binding analysis with Autodock Vina. This work has been cited by <a href="https://ascpt.onlinelibrary.wiley.com/doi/full/10.1002/cpt.1857"> Pfizer </a>, <a href="https://www.researchgate.net/publication/340815389_Hydroxychloroquine_for_the_Treatment_of_COVID-19_Evidence_Possible_Mode_of_Action_and_Industrial_Supply_of_the_Drug">the Italian National Research Council </a>, and at least 24 other researchers. </p>
          </div>
      </div>

    </div>
    <br><br>
  </div>

  <div class="container">
    <div class="section">

      <!--   Icon Section   -->
      <div class="row">
          <div class="icon-block">
	    <h3 class="">Follow Us</h5>

            <p class="light"> Follow us on <a href="https://www.linkedin.com/company/ifftech">Linkedin</a> and <a href="https://github.com/IffTech/"> Github </a> to stay up to date on our progress! 
           </p>
          </div>
      </div>

    </div>
    <br><br>
  </div>

  <footer class="page-footer black">
    <div class="container">
    </div>
    <div class="footer-copyright">
      <div class="container">
	      Copyright 2024 Iff Technologies, LLC. All rights reserved. Get in touch with us at <a href="mailto:contact@iff.bio">contact@iff.bio </a>.
      </div>
    </div>
  </footer>


  <!--  Scripts-->
  <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
  <script src="js/materialize.js"></script>
  <script src="js/init.js"></script>

  </body>
</html>
